manager/ui/war/apiman/config.js-SAMPLE

```diff
@@ -12,7 +12,7 @@ var APIMAN_CONFIG_DATA = {
     "metrics" : true
   },
   "api" : {
-    "endpoint" : "http://localhost:8080/apiman",
+    "endpoint" : "http://localhost:8181/apimanui/apiman",

```

Comment:
Please roll this change back - the sample config should be 8080.  Also not sure why it would ever be /apimanui/apiman?